src/MetaModels/Attribute/Checkbox/Checkbox.php

```diff
@@ -93,35 +93,19 @@ public function getFieldDefinition($arrOverrides = array())
      */
     public function getFilterOptions($idList, $usedOnly, &$arrCount = null)
     {
-        $values = parent::getFilterOptions($idList, $usedOnly, $arrCount);
-        // If not used only or id list, we need to override the handling now.
-        if (!($idList || $usedOnly)) {
-            $values = ['', '1'];
-            if (null !== $arrCount) {
-                $arrCount[''] = 0;
-                $arrCount[1]  = 0;
-                $colName      = $this->getColName();
-                $rows         = $this->getDatabase()->execute(
-                    sprintf(
-                        'SELECT %1$s, COUNT(%1$s) as mm_count FROM %2$s GROUP BY %1$s ORDER BY %1$s',
-                        $colName,
-                        $this->getMetaModel()->getTableName()
-                    )
-                );
-                while ($rows->next()) {
-                    $arrCount[$rows->$colName] = $rows->mm_count;
-                }
-            }
+        
+        $arrOptions = parent::getFilterOptions($idList, $usedOnly, $arrCount);
```

Comment:
1. You are aware that `$arrOptions` is now `[]` when neither `$idList` nor `$usedOnly` is passed?
Before your changes it was `['', '1']`.
2. You killed the whole value counting when neither `$idList` nor `$usedOnly` is passed (which this whole block was for).

Please see the changes in d297bf85461df4e0434ccc4ebc32ad7fbaa82257 and really understand what they do and why they were added before removing all this code and half the functionality along the way.